saturn-datepicker/src/datepicker/datepicker.ts

```diff
@@ -327,6 +332,9 @@ export class SatDatepicker<D> implements OnDestroy, CanColor {
   /** Emits new selected date when selected date changes. */
   readonly _selectedChanged = new Subject<SatDatepickerRangeValue<D>|D>();
 
+  /** The date already selected by the user in range mode. */
+  private beginDateSelected: D;
```

Comment:
According to material style guide this should `private _beginDateSelected: D|null`. If variable is for internal use, it requires `_`. if it is not used in template: `private` + `_`

Reply:
https://github.com/SaturnTeam/saturn-datepicker/pull/68/commits/1548cc6383a0017872bdb95a4431a428360bd04b